playwright/src/test/java/com/microsoft/playwright/junit/TestBrowser2.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.options.BrowserChannel;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+
+import static com.microsoft.playwright.junit.TestOptionsFactories.ChannelOptionsFactory.getBrowserChannelEnumFromEnv;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@UsePlaywright(TestOptionsFactories.ChannelOptionsFactory.class)
+public class TestBrowser2 {
+
+  @Test
+  void shouldSupportDeprecatedChannelEnum(Browser browser) {
+    BrowserChannel channel = getBrowserChannelEnumFromEnv();
+    Assumptions.assumeTrue(channel != null);
```

Comment:
This is too, the `browser` will have been created by the time we check this assumption here. Can we move this call into ChannelOptionsFactory.getOptions and see if the test is skipped if there is no channel? I think it's fine to write this test without the `Browser` fixture (just call the utility methods directly), it would probably make it's purpose more clear. You could still use `Playwright` fixture though.